matter/si91x/siwx917/BRD4343A/autogen/sli_mbedtls_config_autogen.h

```diff
@@ -12,6 +12,7 @@
 #define MBEDTLS_ASN1_WRITE_C
 #define MBEDTLS_BASE64_C
 #define MBEDTLS_CIPHER_C
+#define MBEDTLS_CIPHER_PADDING_PKCS7
```

Comment:
can we revert this change since the define was already present?

Reply:
this was present for only PSA and I moved into the common configuration for both PSA and tinycrypt